src/js/controllers/chatController.js

```diff
@@ -153,7 +153,7 @@ module.exports = ViewController.extend({
                 .then(this._getConversation)
                 .then(this._initFaye)
                 .then(_.bind(function(conversation) {
-                    this.conversationInitiated = true;
+                    this.conversationInitiated = !conversation.isNew();
```

Comment:
Yeah, in this flow, the conversation that you get from `_getConversation` can be a new one (hence `!conversation.isNew()`) if no conversation was started. `_initFaye` is a noop in that case.

Later when you track something, it rechecks the server to see if a conversation was created and if it is the case, it connects to faye. It also fetches the conversation and the messages, so the first whispers might come throught the `fetch` call but everything else will go through faye.